db/models/terminology.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var Terminology = sequelize.define('Terminology', {
+    name: DataTypes.STRING,
```

Comment:
Do we need id in the terminology model?

Reply:
Sequelize will add this automatically when sync'ing the table (along with created/updated fields). We can be more explicit about this.